main.go

```diff
@@ -1,15 +1,21 @@
 package main
 
 import (
+	"fmt"
+	"io/ioutil"
+	"strings"
+
 	"github.com/jiazhen-lin/linebot/api"
```

Comment:
it would be nice if you move line 12~14 before line 8~11

Reply:
Done